setup.py

```diff
@@ -2,12 +2,12 @@
 
 setup(
     name='crema',
-    version='2.31',
+    version='2.32',
```

Comment:
I think minor version update would be correct even though major version update of kafka-python, but does not cause backward incompatible changes in crema in itself